realm/realm-library/src/androidTest/java/io/realm/ManagedRealmCollectionTests.java

```diff
@@ -699,15 +699,19 @@ public void mutableMethodsOutsideTransactions() {
 
     @Test
     public void methodsThrowOnWrongThread() throws ExecutionException, InterruptedException {
+        realm.beginTransaction();
```

Comment:
See commit message f818c1e6335b9d6bf4e89d8bede43398280d86a6

```
    Fix wrong thread test
    
    When call the RealmResults.contains(), the given object must be a
    managed RealmObject, otherwise it will just return false.
```

The temp object contains testing is using is not a valid `RealmObject` before.